### PR TITLE
Allow flipping RGB visualizations on X and Y

### PR DESCRIPTION
**Pull request by axelboc**
We received a request by email to allow flipping RGB visualizations on X and Y.

So far we were always applying a flip on Y. It is still the default, but users can now toggle it off (in addition to flipping on X if they like).

image

### CODE REVIEW

File: apps/storybook/src/RgbVis.stories.tsx
Comment (axelboc) [inline apps/storybook/src/RgbVis.stories.tsx:1]: Documenting `RgbVis` while I'm at it.

**Review by loichuder — APPROVED** [review]
I would have kept `flipY` on by default but ok !

**Comment by axelboc** [comment]
> I would have kept `flipY` on by default but ok !

Yeah, why not.

**Pull request merged**